tools/idf_py_actions/debug_ext.py

```diff
@@ -270,7 +270,7 @@ def generate_gdbinit_rom_add_symbols(target: str) -> str:
             r.append('set confirm on')
             r.append('end')
             r.append('')
-            return os.linesep.join(r)
+            return '\n'.join(r)
```

Comment:
Did `os.linesep` cause any issue when you used it? According to the docs it should be the correct line ending depending on the host OS.